lib/css/modules/_nightMode.scss

```diff
@@ -1472,13 +1471,88 @@ $title-link-visited-color: hsl(0, 0%, 65%);
 		}
 
 		.tfa-login-bottom-panel {
-
 			.tfa-login-having-trouble a {
 				color: $link-color;
 			}
 		}
 	}
 
+	.enable-tfa-modal {
+		.enable-tfa-caption,
+		.modal-header-title,
+		.pref-otp-qr li {
+			color: $color;
+		}
+
+		.otp-secret-info canvas {
+			border: solid 5mm white;
+		}
+
+		.modal-header {
+			background-color: rgb(51, 51, 51);
+			border-color: $border-color;
+			color: $color;
+		}
+
+		.switch-barcode-and-code {
+			color: $link-color;
+		}
+
+		.c-close {
+			background-color: inherit;
+			filter: invert(100%);
+		}
+
+		.enable-tfa-bottom-panel {
+			.enable-tfa-learn-more,
+			.switch-barcode-and-code {
+				a { color: $link-color; }
+			}
+		}
+	}
+
+	.disable-tfa-modal {
+		.modal-header-title,
+		.disable-tfa-caption {
+			color: $color;
+		}
+
+		.c-close {
+			background-color: inherit;
+			filter: invert(100%);
+		}
+
+		.modal-header {
+			background-color: rgb(51, 51, 51);
+			border-color: $border-color;
+			color: $color;
+		}
+
+		.disable-tfa-bottom-panel {
+			.disable-tfa-learn-more {
+				a { color: $link-color; }
+			}
+		}
+	}
+
+	.backup-codes-modal {
+		.modal-header-title,
+		.backup-codes-caption {
+			color: $color;
+		}
+
+		.c-close {
+			background-color: inherit;
+			filter: invert(100%);
+		}
+
+		.modal-header {
```

Comment:
Since `.modal-header`, `.c-close`, and `.modal-header-title` apparently are common for all modals, they can be put in generic `.modal` block.